### ISSUE TITLE
[PROPOSAL] Lower emissions to 250K SBR/day

### ISSUE THREAD

**Issue by rdev42** [saber-hq/governance#55]
## Summary

Full discussion here: https://discord.com/channels/835104494421016586/1314622867878449232/1314622867878449232

During revival phase 1, the SBR emissions in the liquidity gauges was lowered by 50%.

The DAO went from emitting 1,000,000 SBR a day, to 500,000 SBR a day. This seemed to have a positive impact on the SBR markets in hindsight.

**The Proposal**

Reduce liquidity mining emissions in the gauges by 50%. From 500,000 SBR to 250,000 SBR a day. 

**Project Description**

Looking at the current APY based on the SBR emissions, and the price of SBR which has since the initiation of the revival increased by more than 500% it is proposed to further reduce the liquidity mining emissions in the gauges with another 50% to make sure that we keep a balance on APY vs. Liquidity.

**Expectations/Impact**

More balanced SBR emission schedule vs. liquidity in pools supported by the liquidity gauges, and potentially a more stable SBR market (pls note this latter is speculative).

**TLDR:**

Reduce LM emissions in the gauges by 50%
